internal/examples/unittest/closure.go

```diff
@@ -0,0 +1,18 @@
+package unittest
+
+func adder() func(uint64) uint64 {
+	var sum = uint64(0)
+	return func(x uint64) uint64 {
+		sum += x
+		return sum
+	}
+}
+
+func main() {
```

Comment:
Rather than put this test in `main` you should add it to `semantics` and wrap it in a function that actually checks the output matches the expected result. The goose README has some instructions on how to manipulate the auto-generated tests.

Reply:
Looks like you did that - can you delete this from unittest? (We're basically phasing out unit tests in favor of semantics tests, since those actually get run and so we can confirm there's no undefined behavior.)